CacheHelper/CacheHelper.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  CacheHelper.swift
+//  camPod
+//
+//  Created by Janco Erasmus on 2020/04/20.
+//
+
+import Foundation
+
+public class CacheHelper {
+
+    public init() { }
+
+    func saveImage(imageName: String, image: UIImage) {
+        guard let documentsDirectory = FileManager.default.urls(for: .documentDirectory,
+                                                                in: .userDomainMask).first else { return }
+
+        let fileName = imageName
```

Comment:
Please watch out for the spacing in this file.

Reply:
Thanks